src/components/moderation/notify-about-brand-new-users.ts

```diff
@@ -1,10 +1,10 @@
 import * as Discord from "discord.js";
```

Comment:
I think this file I'd prefer to not be in moderation, the moderation/ dir is really meant to be stuff directly issuing and managing moderations

Reply:
okay, moved back out